init-letsencrypt.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v docker-compose -f docker-compose.prod.yml)" ]; then
+  echo 'Error: docker-compose -f docker-compose.prod.yml is not installed.' >&2
+  exit 1
+fi
+
+domains=(example.org www.example.org)
```

Comment:
This should come from a config file. Ideally there's only one place to configure port, domain, etc. Currently it's a bit scattered.

Reply:
Thank you so much! I would try to change the script to use a config file to configure.

Reply:
I have change the script file to use the variable from .env